src/test/java/com/google/api/codegen/testdata/nodejs_doc_message_library.baseline

```diff
@@ -756,21 +756,69 @@ var Book = {
  * @property {Object} field2
  *   This object should have the same structure as [SomeMessage2]{@link SomeMessage2}
  *
+ * @property {number} alignment
+ *   The number should be among the values of [Alignment]{@link Alignment}
```

Comment:
In Node, enum values will be children of the parent namespace/message. So in the case like this, they would be accessed through `SomeMessage.Alignment.GOOD` or `SomeMessage2.Alignment.CENTERED`.

So that would be a bug. That would be `[SomeMessage.Alignment]{@link SomeMessage.Alignment}` -- so then jsdoc can take care of the link address.

You can leave it as-is (but file a bug in toolkit and assign me) or fix it by yourself around `linkForMessage` in `NodeJSGapicContext`.